README.md

```diff
@@ -115,6 +115,7 @@ For syntax, see [lookup.json](./lib/lookup.json), the available attributes are:
 "repo": "https://github.com/pugjs/jade" - Use a different github repo
 "skipAnsi": true             Strip ansi data from output stream of npm
 "script": /path/to/script | https://url/to/script - Use a custom test script
+"scriptWin": \path\to\script | https://url/to/script - Use a custom windows test script
```

Comment:
the tests that we have that run shell scripts and check the return code will fail on windows unless a batch equivalent is added